### PR TITLE
Added support for Linux

### PR DESCRIPTION
**Pull request by GuiEpi**
#### Change in `setup.sh`:
- Creating functions for Linux and Mac
- Change time between runs and notifications in variable

#### Change in `sync.sh`
- Changing the name to `notable-sync.sh` for conflict reasons
- Set git language to English
- Notifications also work

#### Change in `plist.template.xml`
- Adding the new variables. I'm not sure about `ProgramArguments` but I don't have a mac to test...

#### Change in  `readme.md`
- I tried to adapt the readme but I'm not sure I approached it in the best way.

#### Adding a file to create the service
- service.template.ini
- time.template.ini

#### More
On linux it is necessary to launch the script in `sudo` for the creation of systemd files in the root folders. The `-E` argument is therefore necessary for the git commands to work properly.

#### Note
I am not an expert and there may be better ways to do what I did.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,75 +1,133 @@
+#!/bin/bash
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 HAS_GIT=$(which git)
-NOTABLE_DIR=~/.notable
```
Comment (evanshortiss) [inline setup.sh:3]: I don't think this should have been removed?

**Comment by evanshortiss** [comment]
Wow, this is awesome work! Thank you @GuiEpi. I will do a quick test before merging, but it looks great. 

I noticed one variable that seems to be missing when I skimmed the PR just now. LMK if I'm being a dummy and just need to read more closely 😄

File: setup.sh (hunk 1 above)
Comment (GuiEpi) [reply setup.sh:3]: Sure, thanks! I'll fix it.

Comment (GuiEpi) [reply setup.sh:3]: I added it on line **118** and **122** because using `sudo` changes the path of `~/`.

**Comment by GuiEpi** [comment]
No worries, thanks to you, I think the tool is great (it fills a big gap in a very efficient way)!
I just answered and indeed I declare the variable lower. 😁

If there are things you want me to correct in the readme don't hesitate!

**Comment by evanshortiss** [comment]
@GuiEpi haven't had a chance to test, but I see you did indeed declare that var 😁  I'll merge this and make you a collaborator so you can more easily contribute in the future if necessary. Awesome work!

**Pull request merged**